internal/controllers/machine/machine_controller_test.go

```diff
@@ -2869,7 +2781,11 @@ func TestNodeDeletion(t *testing.T) {
 				cluster.DeletionTimestamp = &metav1.Time{Time: deletionTime.Add(time.Hour)}
 			}
 
-			_, err := r.reconcileDelete(context.Background(), cluster, m)
+			s := &scope{
+				cluster: cluster,
+				machine: m,
+			}
+			_, err := r.reconcileDelete(context.Background(), s)
```

Comment:
This is now only a very small part of reconcileDelete, do we have enough coverage for the entire reconcileDelete? (including all the other phases we now run there)

Reply:
reconcileDelete has the same coverage as before, no regressions for this.
the other phases that are run on delete have specific tests for the deletion use case.

Tests like: TestReconcileMachinePhases covers the all of the phases running toghether